### PR TITLE
chore: add configuration for fuzzy metrics

### PR DESCRIPTION
**Pull request by nkoppisetty**
Signed-off-by: Nandita Koppisetty <nandita.iitkgp@gmail.com>

Explain what this PR does.

### CODE REVIEW

File: numaprom/_constants.py [hunk 1]
```diff
@@ -44,6 +44,36 @@
...
+    },
+    "fuzzy_rollouts": {
+        "name": "fuzzy_rollouts",
+        "win_size": 12,
+        "threshold_min": 0.001,
+        "model_name": "ae_sparse",
+        "retrain_freq_hr": 8,
+        "resume_training": "True",
+        "num_epochs": 50,
+        "keys": ["namespace", "name"],
```
Comment (ab93) [inline numaprom/_constants.py:70]: this should have `hash_id` right?

**Review by ab93 — COMMENTED** [review]
Small comment, and how do we get the config through code? Any code change there?

File: numaprom/_constants.py (hunk 1 above)
Comment (nkoppisetty) [reply numaprom/_constants.py:70]: No, the keys wouldn't have hash id, these are for training. I should may be rename it training_keys?

Comment (ab93) [reply numaprom/_constants.py:70]: Maybe; Otherwise might get confusing

**Review by ab93 — APPROVED** [review]

**Pull request merged**